src/simularium/LocalFileSimulator.ts

```diff
@@ -155,13 +134,7 @@ export class LocalFileSimulator implements ISimulator {
     }
 
     public gotoRemoteSimulationTime(time: number): void {
```

Comment:
I can't remember, why are all these functions called "remote" something when it's in the `localFileSimulator`? is it because of the shared interface? It's definitely confusing

Reply:
Yes, remnants of the original code Eric wrote. When I refactored I didn't want to make the diff worse by renaming things so I was saving it for another maintenance PR.  See comment here:
https://github.com/simularium/simularium-viewer/blob/main/src/simularium/ISimulator.ts#L5